R/authors-at-r.R

```diff
@@ -49,11 +49,11 @@ set_author_field <- function(authors, which, field, value) {
 ensure_authors_at_r <- function(obj) {
   if (! obj$has_fields("Authors@R")) {
     stop("No 'Authors@R' field!\n",
-         "You can create one with $add_author")
+         "You can create one with $add_author.\n",
+         "You can also use coerce_authors_at_r(obj) to change Author fields")
```

Comment:
Please remove the 'obj'. It is a method, so it will be called as  `obj@coerce_authors_at_t()`, possibly.

Reply:
Removed.